tests/store/test_rest_store.py

```diff
@@ -232,5 +232,6 @@ def test_requestor(self, request):
                                   message_to_json(expected_message))
             assert result.token == "67890fghij"
 
+
```

Comment:
```suggestion
```

Reply:
Its PEP8 recommendation-Need 2 blank lines after class or function definition.
My pycharm is throwing a warning if I dont.Even mlflow/lint.sh did the same thing when I ran it before pushing the code.